pkg/test/devstack/combodriver_test.go

```diff
@@ -33,7 +32,8 @@ func TestComboDriverSuite(t *testing.T) {
 const exampleText = "hello world"
 
 var testcase scenario.Scenario = scenario.Scenario{
-	ResultsChecker: scenario.FileEquals(ipfs.DownloadFilenameStdout, exampleText),
+	ResultsChecker: scenario.FileEquals(model.DownloadFilenameStdout, exampleText),
+	Contexts:       scenario.CatFileToStdout.Contexts,
```

Comment:
You can remove this line now that WASM executor doens't use `Contexts` any more.

Reply:
```suggestion
```